src/Test/WordPress/HooksTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+
+namespace CF\Test\WordPress;
+
+use \CF\WordPress\Hooks;
+use \CF\Integration\DefaultIntegration;
+use phpmock\phpunit\PHPMock;
+
+class HooksTest extends \PHPUnit_Framework_TestCase
+{
+	use PHPMock;
+
+	protected $hooks;
+	protected $mockConfig;
+	protected $mockDataStore;
+	protected $mockLogger;
+	protected $mockIPRewrite;
+	protected $mockWordPressAPI;
+	protected $mockDefaultIntegration;
+
+	public function setup() {
+		$this->mockConfig = $this->getMockBuilder('CF\Integration\DefaultConfig')
+			->disableOriginalConstructor()
+			->getMock();
+		$this->mockDataStore = $this->getMockBuilder('CF\WordPress\DataStore')
+			->disableOriginalConstructor()
+			->getMock();
+		$this->mockLogger = $this->getMockBuilder('\Psr\Log\LoggerInterface')
+			->disableOriginalConstructor()
+			->getMock();
+		$this->mockIPRewrite = $this->getMockBuilder('\CloudFlare\IpRewrite')
+			->disableOriginalConstructor()
+			->getMock();
+		$this->mockWordPressAPI = $this->getMockBuilder('CF\WordPress\WordPressAPI')
+			->disableOriginalConstructor()
+			->getMock();
+		$this->mockDefaultIntegration = new DefaultIntegration($this->mockConfig, $this->mockWordPressAPI, $this->mockDataStore, $this->mockLogger);
+		$this->hooks = new Hooks($this->mockDefaultIntegration);
+		$this->hooks->setIPRewrite($this->mockIPRewrite);
+	}
+
+	function testInitExecutesFlexibleSSLFix() {
+		$this->mockIPRewrite->method('isCloudFlare')->willReturn(true);
+		$_SERVER['HTTP_X_FORWARDED_PROTO'] = true;
+		$_SERVER['HTTP_X_FORWARDED_PROTO'] = 'https';
+		$this->hooks->init();
+		$this->assertEquals('on', $_SERVER['HTTPS']);
+	}
+
+	function testCloudflareConfigPageCallsAddOptionsPageHookIfItExsits() {
+		$mockFunctionExists = $this->getFunctionMock('CF\WordPress', 'function_exists');
+		$mockFunctionExists->expects($this->once())->willReturn(true);
+		$mock__ = $this->getFunctionMock('CF\WordPress', '__');
```

Comment:
I meant `$mock__ = $this->getFunctionMock('CF\WordPress', '__');`

what is `'__'`?
where is `$mock__` being used?